src/modules/shared/types/media.ts

```diff
@@ -48,7 +48,8 @@ export interface MediaSearchFilter {
 
 export interface MediaInfo {
 	meemoo_fragment_id: string; // Unique id per object
-	schema_identifier: string; // PID (this is not unique per objects)
+	schema_identifier: string;
+	meemoo_identifier: string; // PID (this is not unique per objects)
```

Comment:
I think meemoo_fragment_id no longer exists
So the code with comments becomes:
```typescript
	schema_identifier: string; // Unique id per object
	meemoo_identifier: string; // PID (this is not unique per objects)
```

I can see one more location it is used in the client: src/modules/media/services/media/media.service.ts
But i think this can also be switched to the schema_identifier:
```typescript
		return {
			items: parsed?.hits?.hits.map((item) => ({
				...item._source,
				schema_identifier: item._id,
			})),
			total: parsed?.hits?.total?.value,
			size: size,
			page: page,
			pages: Math.ceil((parsed?.hits?.total?.value || 0) / size),
			aggregations: parsed.aggregations,
		};
```